coolkit/dependencies/dependency.py

```diff
@@ -69,42 +73,47 @@ def set_global_config():
     verify_file('~/.config/coolkit/global_config.py')
     shutil.copy(path_of_default_global_config, path)
 
-def install_dependencies(dependency_map, verbose = False):
-    supported_distros = _get_supported_distros(dependency_map)
-    distro = get_distro(supported_distros)
-    if(not distro and verbose):
-        print_err('unrecognised distro, please contact srbcheema2@gmail.com for full support for your distro',C.R)
-    elif(distro and verbose):
-        print_err('Distro detected to be '+distro+' based',C.G)
 
+def install_dependencies(dependency_map, verbose = False):
     all_installed = True
+    package_manager = get_available_package_manager(dependency_map)
+
+#--handling-absence-of-package-manager--
+    if not package_manager :
+        all_installed = False
+        if verbose :
+            print_err('Error: Unrecognised package manager',C.R)
+            print_err('Please contact srbcheema2@gmail.com for full support for your system.')
+
+            for dependency in dependency_map.keys() :
+                if not is_installed(dependency) :
+                    print_err('Please install ' + dependency + ' dependency manually',C.R)
+#--absence-of-package-manager-handled-
+
+    else :
+        if verbose :
+            print_clr('Package manager detected to be '+ C.E + package_manager , C.G) #distinction of package manager
+
+        for dependency in dependency_map.keys():
+
```

Comment:
remove this empty line.

Reply:
also from else where if some empty lines just leave empty line after a block of code.